src/index.ts

```diff
@@ -40,6 +52,52 @@ const createGenerator: CodegenGeneratorConstructor = (config, context) => {
 			return !!value.mediaType.mimeType.match('\\bjson$')
 		})
 
+		hbs.registerHelper('hasOperationSupportingMultipart', function(value: CodegenOperation | CodegenOperation[]): boolean {
+			if (isCodegenOperation(value)) {
+				return containsMultipartOperation([value])
+			} else {
+				return containsMultipartOperation(value)
+			}
+		})
+
+		hbs.registerHelper('isMultipartSchema', function(value: CodegenSchema): boolean {
+			return !!value.contentMediaType?.mimeType.match('^multipart/.*')
+		})
+
+		hbs.registerHelper('isMetadataSchema', function(value: CodegenSchema): boolean {
+			return value.purpose === CodegenSchemaPurpose.METADATA
+		})
+
+		hbs.registerHelper('getFileUploadProperties', function(properties: CodegenObjectLikeSchemas[]): FileUploadProperty[] {
```

Comment:
What do you think about the name `toFileUploadProperties` instead? It's transforming its input rather than retrieving something.